ksql-engine/src/main/java/io/confluent/ksql/function/udaf/KudafInitializer.java

```diff
@@ -32,10 +32,7 @@ public KudafInitializer(List initialGenericRowColumns) {
 
   @Override
   public GenericRow apply() {
-    List rowColumns = new ArrayList();
-    for (Object obj: initialGenericRowColumns) {
-      rowColumns.add(obj);
-    }
+    List rowColumns = new ArrayList(initialGenericRowColumns);
```

Comment:
creating `rowColumns` via constructor is more performant as list add method dynamically resizes the array